qemu/tests/qemu_guest_agent.py

```diff
@@ -1089,27 +1089,31 @@ class QemuGuestAgentBasicCheckWin(QemuGuestAgentBasicCheck):
     """
     Qemu guest agent test class for windows guest.
     """
+    def __init__(self, test, params, env):
+        QemuGuestAgentBasicCheck.__init__(self, test, params, env)
+        self.gagent_guest_dir = params["gagent_guest_dir"]
+        self.qemu_ga_pkg = params["qemu_ga_pkg"]
+        self.gagent_src_type = params.get("gagent_src_type", "url")
 
     @error_context.context_aware
-    def get_qga_pkg_path(self, test, session, params, vm):
+    def get_qga_pkg_path(self, qemu_ga_pkg, test, session, params, vm):
```

Comment:
This is okay, but next time when we try to extend an API, please add the new arguments in the end of the parameters list of the function.